### ISSUE TITLE
Engine could not create container srcd-cli-gitbase

### ISSUE THREAD

**Issue by gomesfernanda** [src-d/engine#204]
I cannot run `srcd sql`.

**srcd version:**

- srcd cli version: v0.8.0
- docker version: 1.39
- srcd daemon version: v0.8.0

**Mac version:** 10.14.2

First I removed all docker images and was sure no containers were running.

Then I did:
`$ srcd init` and it was ok, installed components and ran them
`$ srcd web parse` and it was ok, installed components and ran them
`$ srcd sql` got different failures, see log below

Then I stopped the containers, and tried running `srcd sql` again
I got failures again, but different ones, see log below

I also checked for port conflicts and didn't find any, nor other babelfish processes running that could conflict.

LOG:

```bash
$ srcd web sql
INFO[0003] this is taking a while, if this is the first time you launch this web client, it might take a few more minutes while we install all the required images
INFO[0003] installing \"bblfsh/bblfshd:v2.11.0-drivers\"
INFO[0027] installed \"bblfsh/bblfshd:v2.11.0-drivers\"
INFO[0027] installed \"bblfsh/bblfshd:v2.11.0-drivers\"
INFO[0027] starting bblfshd daemon
INFO[0027] starting bblfshd daemon
INFO[0029] installing \"srcd/gitbase:v0.18.0\"
INFO[0033] installed \"srcd/gitbase:v0.18.0\"
FATA[0063] could not start gitbase web client at port 8080: rpc error: code = Unknown desc = could not create srcd-cli-gitbase: could not create container srcd-cli-gitbase: context deadline exceeded

$ srcd sql
INFO[0003] this is taking a while, if this is the first time you launch sql client, it might take a few more minutes while we install all the required images
FATA[0030] could not start gitbase: rpc error: code = Unknown desc = could not create srcd-cli-gitbase: could not create container srcd-cli-gitbase: context deadline exceeded

$ srcd stop
INFO[0000] stopping containers...
INFO[0000] removing container srcd-cli-bblfshd
INFO[0000] removing container srcd-cli-daemon

$ srcd sql "SHOW tables;"
INFO[0005] this is taking a while, if this is the first time you launch sql client, it might take a few more minutes while we install all the required images
INFO[0005] starting bblfshd daemon
FATA[0034] could not start gitbase: rpc error: code = Unknown desc = could not create srcd-cli-gitbase: could not create container srcd-cli-gitbase: context deadline exceeded
```

**Comment by marnovo** [comment]
@src-d/applications, please on the bug-fixing side of things give this higher priority as this becomes a blocker next week.

Feel free to reach out to @gomesfernanda to debug it IRL and share the findings here. After we tried we got babelfish (+web) to work, but not gitbase.

**Comment by dpordomingo** [comment]
I wonder if this issue is related to https://github.com/src-d/engine/issues/197, because it returns the same kind of error:

here:
```shell
$ srcd web sql
INFO[0027] installed \"bblfsh/bblfshd:v2.11.0-drivers\"
INFO[0027] starting bblfshd daemon
INFO[0033] installed \"srcd/gitbase:v0.18.0\"
FATA[0063] could not start gitbase web client at port 8080: 
    rpc error: code = Unknown desc = could not create srcd-cli-gitbase:
    could not create container srcd-cli-gitbase: context deadline exceeded
```

there:
```shell
$ srcd web sql
INFO[0599] installed \"bblfsh/bblfshd:v2.11.0-drivers\"
INFO[0599] starting bblfshd daemon
FATA[0599] could not start gitbase web client at port 8080: 
    rpc error: code = Unknown desc = could not create srcd-cli-bblfshd:
    could not create container srcd-cli-bblfshd: context deadline exceeded
```

In both cases, source{d} Engine could not create one needed container: `srcd-cli-gitbase` or `srcd-cli-bblfshd`, because it was received a `context deadline exceeded` error.

**Comment by carlosms** [comment]
Closed by #207.

If the problem persists we will reopen.

**Comment by marnovo** [comment]
Ok, thanks.
